XCUITests/FxScreenGraph.swift

```diff
@@ -435,8 +455,34 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> Scree
         screenState.backAction = navigationControllerBackAction
     }
 
+    map.addScreenState(ChangePasscodeSettings) { screenState in
+        screenState.gesture(forAction: Action.ChangePasscode) { userState in
```

Comment:
This should `transitionTo: LoginSettings` shouldn't it?

Reply:
When the passcode is changed using the ChangePasscode option in Passcode For Loggings screen, it returns there, so it should be to: PasscodeSettings